substrate/frame/revive/src/lib.rs

```diff
@@ -365,6 +372,7 @@ pub mod pallet {
 			type Xcm = ();
 			type RuntimeMemory = ConstU32<{ 128 * 1024 * 1024 }>;
 			type PVFMemory = ConstU32<{ 512 * 1024 * 1024 }>;
+			type ChainId = ConstU64<{ 0 }>;
```

Comment:
You mind setting it to 42 here, I need to reconfigure my Metamask extension otherwise 🙃

Reply:
What do you mean by extensions? Shouldn't Metamask just use an RPC to query to chain id?

Reply:
Also this ChainId is not really used. It is just the default derive that can be used for tests. But it is overriden in tests.

Reply:
if you configure a local network with metamask, you need to specify the chain_id, you can update it later on, but from what I experienced there is a bug and it still use the old value, so you delete your custom network  and recreate it.

Reply:
42 is [already taken](chainlist.org) and I thought this should either be zero or something that's not taken yet? (Something that's not taken yet could however get taken later on). WDYT?